crates/chronicle/src/bootstrap/cli.rs

```diff
@@ -912,7 +912,7 @@ impl SubCommand for CliModel {
         let mut app = Command::new("chronicle")
             .version(self.short_version.as_str())
             .long_version(self.long_version.as_str())
-            .author("Blockchain technology partners")
+            .author("BTPWorks")
```

Comment:
```suggestion
            .author("BTP Works")
```
is how I've seen it to date but I may have missed a memo.

Reply:
The legal entity here is still "Blockchain Technology Partners"  so it should stay the way it is.